client/my-sites/plans/jetpack-plans/product-store/items-list/all-items.tsx

```diff
@@ -45,29 +48,46 @@ export const AllItems: React.FC< AllItemsProps > = ( {
 					const isItemSuperseded = isSuperseded( item );
 					const isItemDeprecated = isDeprecated( item );
 					const isItemIncludedInPlanOrSuperseded = isIncludedInPlanOrSuperseded( item );
+					const isItemIncludedInPlan = isIncludedInPlan( item );
 
 					const isCtaDisabled =
-						( isItemOwned || isIncludedInPlan( item ) ) && ! isUserPurchaseOwner( item );
+						( isItemOwned || isItemIncludedInPlan ) && ! isUserPurchaseOwner( item );
 
 					const ctaLabel = getCtaLabel( item );
 
 					const hideMoreInfoLink =
 						isItemDeprecated || isItemOwned || isItemIncludedInPlanOrSuperseded;
 
+					const price = (
+						<ItemPrice
+							isIncludedInPlan={ isItemIncludedInPlan }
+							isOwned={ isItemOwned }
+							item={ item }
+							siteId={ siteId }
+						/>
+					);
+
+					const description = (
+						<>
+							{ item?.shortDescription || item.description } <br />
+							{ ! hideMoreInfoLink && (
+								<MoreInfoLink onClick={ onClickMoreInfoFactory( item ) } item={ item } />
+							) }
+						</>
+					);
+
 					return (
 						<SimpleProductCard
```

Comment:
Now that this card becomes a generic one. Maybe we can rename it to SimpleItemCard or just SimpleCard.

Reply:
Oh, that's a good idea. Let me do that.

Reply:
Done. Thanks